docs/bonus_guides/learning_elixir.md

```diff
@@ -4,7 +4,7 @@ Here is a selected list of resources to help with learning Elixir and Erlang.
 
 ## Elixir
 - [Elixir Guides](http://elixir-lang.org/getting-started/introduction.html)
-- [Official Documentation](http://elixir-lang.org/docs/stable/elixir)
+- [Official Documentation](https://hexdocs.pm/elixir/1.5.1/api-reference.html)
```

Comment:
Linking to https://hexdocs.pm/elixir/1.5.1 redirects to https://hexdocs.pm/elixir/1.5.1/Kernel.html which didn't seem super helpful, so that's why the link is to api-reference